apps/gsuite/mail_syncer/tests/apiclient_utils_tests.py

```diff
@@ -0,0 +1,275 @@
+from django.conf import settings
+from django.contrib.auth.models import Group
+from django.test import TestCase, override_settings
+from django_dynamic_fixture import G
+from googleapiclient.errors import HttpError
+from mock import patch
+
+from apps.authentication.models import OnlineUser
+from apps.gsuite.mail_syncer.tests.test_utils import create_http_error
+from apps.gsuite.mail_syncer.utils import (_get_excess_users_in_g_suite,
```

Comment:
Do we need to test "private" functions?

IMO they should be tested as part of their public functions, so that we are free to change the implementation details easily.